src/_js/lib/Page.js

```diff
@@ -9,4 +9,8 @@ export const logPageview = function() {
     url: canonicalUrl(location.search)
   });
   window.ra.page();
+  if (window.gtag) {
```

Comment:
I don't think this is necessary. The tracker code does the config stuff on the first page load and that persists until you navigate off-site or the page refreshes.

Reply:
we don't load the tracker code until the user consents, so window.gtag won't exist if they navigate to a dynamic page and window.gtag() will throws a JS error..

Reply:
Ah, forgive me. I've been away from this one for too long.